src/Resources/views/admin/bulk-upload/data-flow-profile/index.blade.php

```diff
@@ -4,10 +4,18 @@
     {{ __('bulkupload::app.admin.bulk-upload.data-flow-profile.index') }}
 @stop
 
+@push('css')
+    <style>
+        .ms-15 {
```

Comment:
Please use the scss files https://github.com/bagisto/bagisto-bulk-upload/tree/master/src/Resources/assets/sass